Sources/ComposableArchitecture/Internal/DefaultSubscript.swift

```diff
@@ -1,7 +1,7 @@
-final class DefaultSubscript<Value>: Hashable {
-  var value: Value
+final class DefaultSubscript<Value: Sendable>: Hashable, Sendable {
```

Comment:
We know we want to remove these helpers in the future, but since they're use by `store.scope` let's split off a non-sendable version for that.